test/integration_testcase.go

```diff
@@ -118,7 +140,14 @@ func getFile(path string) (bool, string, error) {
 	if err != nil {
 		return false, "", nil
 	}
-	f, err := ioutil.ReadFile(path)
+
+	// Ensure it's the absolute, symlink-less path we're returning
+	abs, err := filepath.EvalSymlinks(path)
+	if err != nil {
+		return false, "", nil
+	}
+
+	f, err := ioutil.ReadFile(abs)
```

Comment:
tbh, I am unsure if that symlink was needed so will remove it for the moment. As mentioned in the above comment I removed it from a different file as it caused the tests to fail when no `lock.json` was in the `final` dir.